cabal-testsuite/PackageTests/MultiRepl/EnabledBadClosure/cabal.test.hs

```diff
@@ -0,0 +1,11 @@
+import Test.Cabal.Prelude
+
+main = do
+  cabalTest $ do
+    -- MP: TODO: This should query Cabal library version
```

Comment:
I think the failures are related to this TODO.
The failure seems to be when parsing the `custom-setup` stanza, and we see in the failure message that the `Cabal` library being used is 2.4.

```
Warning: foo.cabal:0:0: Unsupported cabal-version. See https://github.com/haskell/cabal/issues/4899.
-----END CABAL OUTPUT-----
CallStack (from HasCallStack):
  die', called at libraries/Cabal/Cabal/Distribution/Parsec/ParseResult.hs:184:13 in Cabal-2.4.0.1:Distribution.Parsec.ParseResult
  parseString, called at libraries/Cabal/Cabal/Distribution/Parsec/ParseResult.hs:169:5 in Cabal-
```

Reply:
I think we can do with `skipUnlessGhcVersion ">= 8.8"`